pkg/clean/clean.go

```diff
@@ -25,19 +27,35 @@ var log = logger.Log
 // including security policy, static route, subnet, subnet port, subnet set, vpc, ip pool, nsx service account
 // it is usually used when nsx-operator is uninstalled and remove all the resources created by nsx-operator
 // return error if any, return nil if no error
+// the error roughly are:
```

Comment:
Do we need to add error code for these cases, or do they return static error messages, especially should be user friendly for the above 3 cases

Reply:
The error code should not be here - they should be in `main.go`. In this case, specific error types should be returned.

Reply:
ok